build/yamls/antrea-windows.yml

```diff
@@ -204,6 +204,8 @@ spec:
           name: antrea-agent-windows
         - mountPath: /host/k/antrea/
           name: host-antrea-home
+        - mountPath: /host/var/log/antrea/
```

Comment:
If we mount host's `/var/log/antrea` to container's `/host/var/log/antrea/`, why does the configuration and `antreaWellKnownLogDir` still point to `/var/log/antrea/`?

Reply:
IIRC the Agent actually runs as a process on the Windows host, even though its lifecycle is managed by a DaemonSet using `wins`.

But now that I think of it, is this mount even required? @GraysonWu?

Reply:
You are right. No need to mount. Removed.